src/js/extension-anchor-preview.js

```diff
@@ -133,13 +134,36 @@ var AnchorPreview;
             this.instance_handleAnchorMouseout = null;
         },
 
+        getClosestTag : function(el, tag) {
```

Comment:
I guess that function will need to be in the Util class instead.
Also, don't forget to add a comment above the function to tell what it should do.